src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -470,6 +489,43 @@ MethodSymbol getTopLevelMethod(MethodSymbol method)
                 }
                 return null;
             }
+
+            // Returns true if the nullable state from the operand of the conversion
+            // can be used as is, and we can create a slot from the conversion.
+            bool isSupportedConversion(Conversion conversion, BoundExpression operandOpt)
```

Comment:
>isSupportedConversion [](start = 17, length = 21)

Thanks for the comment. Perhaps rename to `conversionIntroducesSlot` or something along those lines?

Reply:
I think I prefer `isSupportedConversion`. It seems sufficiently descriptive since we're in a `MakeSlot` method.

---
In reply to: [249162797](https://github.com/dotnet/roslyn/pull/32612#discussion_r249162797) [](ancestors = 249162797)